.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,34 @@
 ---
-title: 'Bug report'
+name: 🐛 Bug report
+about: Something's not quite right? You're in the right place!
 labels: user report
 ---
 
-### 🐛 Bug Report
+# 🐛 Bug Report
```

Comment:
This is going to be massive text! I recommend making this ### and all other fields ####. This applies to the feature request template as well.